reference/api/search.md

```diff
@@ -460,7 +460,17 @@ Optionally, you can indicate a custom crop length for any attributes given to `a
 
 Instead of supplying individual attributes, you can provide `["*"]` as a wildcard: `attributesToCrop=["*"]`. This causes `_formatted` to include the cropped values of all attributes present in [`attributesToRetrieve`](#attributes-to-retrieve).
 
-**Meilisearch crops around the first occurrence of any one of the terms present in the search query.** If Meilisearch does not find any query terms in a field, cropping begins at the first word in that field.
+#### Cropping rules
```

Comment:
"Rules" makes me think of ranking rules, linter rules, things that are customizable by the user. I would prefer something like "Cropping algorithm" or "Cropping explanation".